### DISCUSSION TITLE
AliyunIOT trigger for function computer

### DISCUSSION THREAD

**Discussion by pw8oktas** [Serverless-Devs/Serverless-Devs#652] [Ideas]
Hi guys,

I am trying to use serverless dev to deploy a function compute, that is triggered by Aliyun IOT.
This feature is support and is available for a while via enterprise instances.

See link below: in chinese  (that is the most up to date version)

However, the current serverless s.yaml does not support the IOT trigger for function compute.

I think this is a very core feature, consider the use event driven servlerless app, via mqtt communication. This is a very common feature in other provider.

is there any chance serverless-devs will update and support IOT trigger . 

Thanks

Pat

**Comment by heimanba** [comment]
hi,we support iot trigger by  [eventbridge](https://eventbridge.console.aliyun.com/), you can choose iot trigger on console 
![image](https://user-images.githubusercontent.com/5129967/221062552-6448c950-044d-4f14-a064-c1cbc4481bfb.png)
and export config 
![image](https://user-images.githubusercontent.com/5129967/221062608-fdcd9ab9-05e4-405d-871b-d81389432040.png)

**Comment by pw8oktas** [comment]
hi, so the ros template or serverless dev does not support configuration via s.yaml?

the only way to do this is configuration via console?

Pat
